### ISSUE TITLE
Slideshow View Unique Edit

### ISSUE THREAD

**Issue by cambelem** [AppStateESS/slideshow#32]
Currently, when we click on edit for a slide show, we are not going to a unique 'edit' state for that slideshow. All slideshows go to the same edit state.

# Note
This should be rather simple to complete, we just need to pass an id around with the request and pass it to the load. I have implemented a part of this in the editJsonCommand however, the id needs to be pulled from the request, AND I currently haven't pushed this part of the method up yet as it interferes with what I am currently working on.
# TL;DR
 This shouldn't be picked up until [load](https://github.com/AppStateESS/slideshow/issues/34) is complete.
